frontend/src/hooks/useValidateLoginForm.ts

```diff
@@ -0,0 +1,57 @@
+import { useVuelidate } from '@vuelidate/core';
+import { email, required, minLength } from '@vuelidate/validators';
+import { computed } from 'vue';
+
+import type { ComputedRef, Ref } from 'vue';
+import type { 
+  ErrorObject,
+  Validation, 
+  ValidationRuleWithoutParams, 
+  ValidationRuleWithParams 
+} from '@vuelidate/core';
+
+interface ILoginState {
+  email: string,
+  password: string
+}
+
+interface ILoginRules {
+  email: {
+    required: ValidationRuleWithoutParams<any>,

```

Comment:
Replace any with the actual type.
use _any_ should be the last resort option

Reply:
Removed these types (they were from Vuelidate lib.)